### DISCUSSION TITLE
Retweet Problemi

### DISCUSSION THREAD

**Discussion by maestrosoner** [IBAX-io/go-ibax#1779] [Soru Cevap - Türkçe] [answered]
Ibax Retweetlerini paylaşırken sorun yaşamıyorduk fakat bugün Retweet yapmak istediğimde daha sonra tekrar deneyin yazıyor.

**Comment by Bulent3407** [comment, marked as answer]
Evet malesef beklemedeyiz

**Reply by ahmederoni** [reply]
Bu konuyla ilgili çok şikayet var ama ilgili kimse yok,telegram kanallarında admin bile yok aklıma kötü kötü şeyler geliyor ama durduk yere  kimseyi strese sokmayım yarın ola hayrola

**Reply by maestrosoner** [reply]
Bir çok kişi bu sorunu yaşamış maalesef. Bekleyip göreceğiz artık. Yeni bir sistem güncellemesi muhtemel.

**Comment by coinhakann** [comment]
Sorunun çözümü yok değil mi henüz?

**Reply by maestrosoner** [reply]
Henüz yok.

**Reply by Yusufknk** [reply]
Retweet görevleri için belirli sayıda API kontenjanı var bu kontenjan son retweet görevlerinde az sayıda kaldığı için ilk yapanlar doldurdu. Alınan hataların ve çoğu kişinin yapamamasinin nedeni bu

**Comment by Yusufknk** [comment]
Retweet görevleri için belirli sayıda API kontenjanı var bu kontenjan son retweet görevlerinde az sayıda kaldığı için ilk yapanlar doldurdu. Alınan hataların ve çoğu kişinin yapamamasinin nedeni bu
